hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestSnapshotDeletingService.java

```diff
@@ -471,4 +471,3 @@ private boolean assertTableRowCount(int expectedCount,
     return count == expectedCount;
   }
 }
-
```

Comment:
[It does](https://github.dev/apache/ozone/blob/c12239078e9e8799159f5a01ac418bda07b0dced/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestSnapshotDeletingService.java):

<img width="462" alt="eofnewline" src="https://github.com/apache/ozone/assets/50227127/14122c0d-856d-4eb6-9a57-6d8972f77497">

Otherwise it would show a symbol like this at EOF: https://stackoverflow.com/q/65300309

You can say that is a GH PR diff display issue.

Reply:
Ah, I see. I think you remove an extra line and got confused.